pyboy/plugins/game_wrapper_metroid_ii.py

```diff
@@ -0,0 +1,191 @@
+#
+# License: See LICENSE.md file
+# GitHub: https://github.com/Baekalfen/PyBoy
+#
+__pdoc__ = {
+    "GameWrapperMetroidII.cartridge_title": False,
+    "GameWrapperMetroidII.post_tick": False,
+}
+
+import pyboy
+from pyboy import utils
+from pyboy.utils import WindowEvent
+# for memory
+from pyboy.utils import bcd_to_dec
+
+from .base_plugin import PyBoyGameWrapper
+
+logger = pyboy.logging.get_logger(__name__)
+
+
+class GameWrapperMetroidII(PyBoyGameWrapper):
+    """
+    This class wraps Metroid II, and provides easy access for AIs.
+    If you call `print` on an instance of this object, it will show an overview of everything this object provides.
+    """
+    cartridge_title = "METROID2"
+
+    def __init__(self, *args, **kwargs):
+        self._game_over = False        
+        # may need to change game area section. Copied from the kirby code
+        super().__init__(*args, game_area_section=(0, 0, 20, 16), game_area_follow_scxy=True, **kwargs)
```

Comment:
Could you add all the attributes you've added here and give them a default value? And underneath each attribute add a docstring?